adhocracy4/comments/models.py

```diff
@@ -45,6 +45,10 @@ class Comment(base.UserGeneratedContentModel):
         null=True,
         editable=False
     )
+    is_recommended = models.BooleanField(
```

Comment:
wrong wording ;)

Reply:
But that's why WIP!

Reply:
ah alright, I thought wenn you added me to reviewers you wanted me to actually look into it :)